std/net/curl.d

```diff
@@ -3973,7 +3969,7 @@ struct Curl
 {
     alias OutData = void[];
     alias InData = ubyte[];
-    bool stopped;
```

Comment:
No need, I didn't see the function. Just mark it `@property` and it should be good.